CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - More detailed and sophisticated search space user guide
 - Upgraded syntax to Python 3.9
+- Use `tox-uv` in pipelines
```

Comment:
Regarding your question whether "What do we do": Let's just wait with this PR

Reply:
@Scienfitz, @AVHopp. Seems this issue is now resolved. Can you both give me a quick "thumbs up" reaction, then I'll merge

Reply:
gucci